og.og_permissions.yml

```diff
@@ -2,10 +2,11 @@ update group:
   title: 'Edit group'
   description: 'Edit the group. Note: This permission controls only node entity type groups.'
   'default role':
-    - OG_ADMINISTRATOR_ROLE
```

Comment:
Off the top of my head not sure. But this VERSION string is hard coded in
the module discovery somewhere.
On 29 Nov 2015 17:59, "Amitai Burstein" <notifications@github.com> wrote:

> In og.og_permissions.yml
> <https://github.com/amitaibu/og/pull/48#discussion_r46095631>:
>
> > @@ -2,10 +2,11 @@ update group:
> >    title: 'Edit group'
> >    description: 'Edit the group. Note: This permission controls only node entity type groups.'
> >    'default role':
> > -    - OG_ADMINISTRATOR_ROLE
>
> yeah, it's a good idea, because also if I remove the custom define we have
> in the unit test, it will fail due to its use in
> \Drupal\og\OgPermissionHandler::buildPermissionsYaml
>
> Any pointer to those 'special' strings?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/amitaibu/og/pull/48/files#r46095631>.
>